src/index.html

```diff
@@ -11,7 +11,172 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    onsubmit="onSubmit()"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
```

Comment:
Fix all issues like this 🙏. I see more than one similar mistake

```suggestion
    <form
      onsubmit="onSubmit()"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```